### PR TITLE
upgrade sbt for language server features [BW-485]

### PR DESCRIPTION
**Pull request by panentheos**
This upgrades sbt to the latest version and adds some `.gitignore` entries related to the Metals language server. This enables code navigation in non-intellij editors using Metals and the recent BSP support in sbt (tested in vscode, but should work in any editor supported by Metals)

@Qi77Qi FYI

- [ ] **Submitter**: Include the JIRA issue number in the PR description
- [ ] **Submitter**: Check that the **Product Owner** has signed off on any user-facing changes
- [ ] **Submitter**: Make sure Swagger is updated if API changes
  - [ ] **...and Orchestration's Swagger too!**
- [ ] **Submitter**: If updating admin endpoints, also update [firecloud-admin-cli](https://github.com/broadinstitute/firecloud-admin-cli)
- [ ] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [ ] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [ ] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [ ] **Submitter**: Database checks:
  * If PR includes new or changed db queries, include the explain plans in the description
  * Make sure liquibase is updated if appropriate
  * If doing a migration, take a backup of the
  [dev](https://console.cloud.google.com/sql/instances/terraform-qfarbdq3lrexxck5htofjs5z6m/backups?project=broad-dsde-dev&organizationId=548622027621)
  and
  [alpha](https://console.cloud.google.com/sql/instances/terraform-r4caezzc35c4tb7pgdhwkmme4y/backups?project=broad-dsde-alpha&organizationId=548622027621)
  DBs in Google Cloud Console
- [ ] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [ ] Tell your tech lead (TL) that the PR exists if they want to look at it
- [ ] Anoint a lead reviewer (LR). **Assign PR to LR**
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [ ] **TL** sign off
- [ ] **LR** sign off
- [ ] **Assign to submitter** to finalize
- [ ] **Submitter**: Verify all tests go green, including CI tests
- [ ] **Submitter**: Squash commits and merge to develop
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Verify swagger UI on dev environment still works after deployment
- [ ] **Submitter**: Inform other teams of any API changes via Slack and/or email
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

**Review by Qi77Qi — COMMENTED** [review]
👍  :shipit:

**Comment by Qi77Qi** [comment]
there's also https://github.com/broadinstitute/rawls/blob/develop/automation/project/build.properties if you're up for updating too.

If you update the docker file for `automation`, I think we should update [this](https://github.com/broadinstitute/rawls/blob/develop/automation/Dockerfile-tests#L1) to  https://github.com/DataBiosphere/leonardo/blob/develop/Dockerfile-tests#L1 .. I think this will make ci a bit faster cuz the docker image for automation will come with the correct sbt version

File: automation/project/build.properties [hunk 1]
```diff
@@ -1 +1 @@
-sbt.version = 1.4.0
\ No newline at end of file
+sbt.version = 1.4.6
```
Comment (cjllanwarne) [inline automation/project/build.properties]: The other files seem to be newline-terminated. I'd do the same here for consistency (and to play nicely with command line commands)

**Review by cjllanwarne — APPROVED** [review]
Looks good to me as long as the tests are all still happy 👍

**Pull request merged**